ndcube/ndcube.py

```diff
@@ -408,46 +412,47 @@ def axis_world_coords(self, *axes, edges=False, wcs=None):
         return tuple(axes_coords[i] for i in object_indices)
 
     @utils.misc.sanitise_wcs
-    def axis_world_coords_values(self, *axes, edges=False, wcs=None):
+    def axis_world_coords_values(self, *axes, pixel_corners=False, wcs=None):
         """
         Returns WCS coordinate values of all pixels for desired axes.
 
         Parameters
         ----------
-        axes: `int` or `str`, or multiple `int` or `str`
+        axes: `int` or `str`, or multiple `int` or `str`, optional
             Axis number in numpy ordering or unique substring of
-            `~ndcube.NDCube.wcs.world_axis_physical_types`
+            `~ndcube.NDCube.world_axis_physical_types`
```

Comment:
There is no `~ndcube.NDCube.world_axis_physical_types`, right.  Should this be `~ndcube.NDCube.array_axis_physical_types`?  If not, isn't what's already there correct?

Reply:
I do not remember making this change :thinking: 

```suggestion
            `~ndcube.NDCube.wcs.world_axis_physical_types`
```